server/webapp/WEB-INF/rails.new/spec/new_javascripts/models/pipeline_configs/jobs_spec.js

```diff
@@ -68,6 +68,10 @@ define(['lodash', "models/pipeline_configs/jobs", "string-plus"], function (_, J
       expect(job.elasticProfileId()).toEqual('docker.unit-test');
     });
 
+    it('should be assigned to elastic profile', function () {
+      expect(job.isAssignedToElasticAgent()).toBeTruthy();
```

Comment:
Where's the negative test? Also, since the method says `is` it'd be better to test for `true` and `false` instead of `truthy` and `falsy`